test/junit/scala/concurrent/duration/DurationTest.scala

```diff
@@ -0,0 +1,27 @@
+package scala.concurrent.duration
```

Comment:
@Jasper-M If you feel adventurous, I guess you could tackle porting the duration-tck to JUnit :)